lib/linter.js

```diff
@@ -260,6 +260,7 @@ class Linter {
     return currentSource;
   }
 
+  // checks to see if config is being overridden so it doesn't remove something that should stay
```

Comment:
I'm not sure what this comment means per se, and also don't think it's necessary. Can we remove it?

Reply:
yes, it's just there for me to remember. I'll remove it before we ship it.